src/packages/@ncigdc/utils/trace.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default C => p => (
+  console.info(
+    `%ctracing ${C.displayName} props:\n==========================`,
+    'color: rgb(22, 138, 96);font-weight: bold;',
+  ), // eslint-disable-line
+  console.log(p), // eslint-disable-line
+  <C {...p} />
```

Comment:
How do you feel about this?

```
C(p)
```

Also let's avoid single character variable names eh? Particularly if they abbreviate to CP